src/site/pages/upgrade-guide.md

```diff
@@ -0,0 +1,228 @@
+# Upgrading to PatternFly 4
+
+Use the following steps to upgrade from PatternFly 3 to PatternFly 4.
+
+**Before you begin**
+
+- **Does you application require IE11 support?**
+   - If the answer is yes, you can't upgrade to PatternFly 4. We're sorry!
+- **Do you know what your build process is?**
+  - Steps and requirements differ for basic HTML, Gulp, and Webpack, so be prepared with that information going in.
+- **Do you need to run PatternFly 3 and PatternFly 4 together?**
+  - If the answer is yes, use the following steps (or run the following commands?) to update your configuration in the `src/patternfly/utilities/variables.scss` file to state `false`:
```

Comment:
Make sure to take out the piece in the parentheses where I say (or run the following commands?) :)

Reply:
Gah, completely missed that :(